src/main/java/io/redisearch/client/Client.java

```diff
@@ -847,7 +847,24 @@ public Long addSuggestion(Suggestion suggestion, boolean increment) {
                 return getSuggestionsWithScores(args);
         }
     }
-    
+
+    @Override
+    public Long deleteSuggestion(String entry) {
+        List<String> args = new ArrayList<>();
```

Comment:
@tgrall can please just replace the ArrayList with a `String[] args = new String[2]`?

Reply:
Shame on me.... ;) 
Done